client/components/data/query-top-posts/README.md

```diff
@@ -0,0 +1,65 @@
+Query Top Posts
```

Comment:
Hi @mattsherman! I just pushed an update. It adds a new section to the live docs, “Data Components”, as there was nothing for this kind of component. You will find there a configuration tool with some options to test the behavior of the `QueryTopPosts` component. Please tell me if that’s what you were expecting :-)